docs/examples/pyangbind/index.md

```diff
@@ -25,6 +25,8 @@ pip3 freeze | grep pyang
 
 ## Get YANG modules
 
+We need YANG modules
```

Comment:
incomplete?

Reply:
just added `We need YANG modules so we can use Pyang and Pyangbind`